### ISSUE TITLE
Build failing with last update

### ISSUE THREAD

**Issue by erasmo-marin** [catamphetamine/print-error#3]
Hi, thanks for the update but I'm getting this on last version:

```
Failed to compile.

Module not found: Error: Can't resolve './source/parse' in '/Users/erasmo/projects/fx-cm-2.0/node_modules/print-error'
```
it's trying to resolve from source

**Comment by catamphetamine** [comment]
Oh, will fix now

**Comment by catamphetamine** [comment]
Fixed and renamed `ERROR_STACK_STYLE` to `HTML_ERROR_STACK_STYLE`.
Published `print-error@0.1.17`.
